app/forms/base_form.rb

```diff
@@ -0,0 +1,52 @@
+class BaseForm
+  extend ActiveModel::Naming
+  include ActiveModel::Model
+  include ActiveModel::Serialization
+
+  delegate :attributes=, :attributes, to: :attachments, prefix: true
+  delegate :valid?, to: :object
+  delegate :to_param, :errors, :persisted?, to: :object
+
+  attr_accessor :object, :current_user
+
+  def initialize(object, user, params = {})
+    @object = object
+    @current_user = user
+    @attachments_params = uploaded_attachments params
+    @params = params
+  end
+
+  def save
+    return false unless valid?
+
+    object.save
+    save_attachments if attachments_params
+  end
+
+  def update
+    return false unless current_user == object.user
+
+    object.update params
+    save_attachments if attachments_params
+    true
```

Comment:
???

Reply:
Need to return something like `true`, because `save_attachments` return `[]`.
Now I return `object`.